components/blogDetail/content.tsx

```diff
@@ -0,0 +1,350 @@
+                  color="text.secondary"
+                  sx={{
+                    fontSize: "14px",
+                    textAlign: "left",
+                    color: "#515151",
+                  }}
+                >
+                  {blogcontent.pharagraph.imageContentText}{" "}
+                  <Link href="" sx={{ color: "#515151" }}>
+                    {blogcontent.pharagraph.imageContentsource}
+                  </Link>
+                </Typography>
+              </Grid>
+              {/* Grid item for blockquote */}
+              <Grid item xs sx={{ alignItems: "center", textAlign: "center" }}>
+                {/* Blockquote */}
+                <blockquote
+                  style={{
+                    color: "#11142D",
+                    fontSize: "24px",
+                    textAlign: "justify",
+                    borderLeft: "0.2em solid",
+                    borderColor: "#6941C6",
+                    margin: "1.5em 10px",
+                    padding: "0.5em 10px",
+                    fontStyle: "italic",
+                    fontWeight: "normal",
+                  }}
+                >
+                  {blogcontent.pharagraph.blockquoteText}
+                  <br />
+                  <cite
+                    style={{
+                      color: "#515151",
+                      fontSize: "15px",
+                      fontStyle: "normal",
+                    }}
+                  >
+                    -{blogcontent.pharagraph.blockquoteCite}
+                  </cite>
+                </blockquote>
+  
+                {/* Body paragraphs */}
+                <Typography
+                  gutterBottom
+                  sx={{
+                    color: "#515151",
+                    fontSize: "20px",
+                    textAlign: "justify",
+                  }}
+                >
+                  {blogcontent.pharagraph.bodyParagraph}
+                  <br />
+                  <br />
+                  {blogcontent.pharagraph.bodyParagraph1}
+                  <br />
+                  <br />
+                </Typography>
+                {/* Second paragraph title */}
+                <Typography
+                  variant="body2"
+                  color="text.secondary"
+                  sx={{
+                    fontSize: "30px",
+                    textAlign: "left",
+                    fontWeight: "bold",
+                    color: "#11142D",
+                  }}
+                >
+                  {blogcontent.pharagraph.pTitle2.title}
+                </Typography>
+                {/* Second paragraph content */}
+                <Typography
+                  gutterBottom
+                  sx={{
+                    color: "#515151",
+                    fontSize: "20px",
+                    textAlign: "justify",
+                  }}
+                >
+                  {blogcontent.pharagraph.pTitle2.paragraph1}
+                  <br />
```

Comment:
Don't use <br> tags . Use padding and margins 
https://mui.com/system/spacing/

Reply:
**Done.**